src/biokbase/narrative/jobs/job.py

```diff
@@ -18,18 +18,19 @@
 COMPLETED_STATUS = "completed"
 TERMINAL_STATUSES = [COMPLETED_STATUS, "terminated", "error"]
 
-EXCLUDED_JOB_STATE_FIELDS = [
+JOB_INIT_EXCLUDED_JOB_STATE_FIELDS = [
```

Comment:
This PR comprises a few minor changes to what is included in the `jobState` object returned by the backend when a `job_status` request is made -- basically just removing fields that aren't used by the frontend.